server/app/assets/javascripts/admin_programs.ts

```diff
@@ -21,6 +21,32 @@ class AdminPrograms {
       },
     )
   }
+
+  // When the common intake checkbox is selected,
+  // the program category checkboxes should be disabled and unchecked.
+  static attachCommonIntakeChangeListener() {
+    addEventListenerToElements('#common-intake-checkbox', 'click', () => {
+      const commonIntakeCheckbox = <HTMLInputElement>(
+        document.querySelector('#common-intake-checkbox')
+      )
+
+      const programCategoryCheckboxes = document.querySelectorAll(
+        '[id^="checkbox-category"]',
+      )
+
+      programCategoryCheckboxes.forEach((checkbox) => {
+        const category = checkbox as HTMLInputElement
+
+        if (commonIntakeCheckbox.checked) {
+          category.disabled = true
+          category.checked = false
+        } else {
+          category.disabled = false
```

Comment:
should we add catogory.checked=true here?

Reply:
If we set `category.checked=true`, it will select all the categories when it isn't an intake form.  We should let the user determine which categories should be applied to the program.